tests/test_shseq.nim

```diff
@@ -0,0 +1,24 @@
+{.used.}
+
+import unittest2, ../metrics/shseq
+
```

Comment:
This file needs some cross-thread tests.

Reply:
[2f53329](https://github.com/status-im/nim-metrics/pull/97/commits/2f533298bd198b9cab353e0381958d658d39ef9d)

Reply:
[6674364](https://github.com/status-im/nim-metrics/pull/97/commits/66743644b641b991d3410e4902a6161c2984d931)